docs/en/operation/Basic-Logging.md

```diff
@@ -28,8 +28,8 @@ Files suffixed with `.log` like `master.log` or `worker.log` are generated by `l
 logging the events that Alluxio system is recording through JVM.
 These files are typically the main target for users to investigate logs.
 Files suffixed with `.out` like `master.out` or `worker.out` are the redirection of
-`stdout` and `stderr` of the corresponding process,
-typically empty unless fatal error happens (e.g., killed by operating system).
+`stdout` and `stderr` of the corresponding process. 
+Fatal error messages (e.g., killed by the OS), will most likely go to these files.  
```

Comment:
The .out file will likely contain many gRPC errors like connection reset etc. Most likely they will not be empty in a real environment (and instead be big).